EventsExpress/Controllers/AuthenticationController.cs

```diff
@@ -41,27 +54,24 @@ IAuthService authSrv
         [AllowAnonymous]
         [HttpPost("[action]")]
         [Produces("application/json")]
-        public IActionResult Login(LoginDto authRequest)
+        public async Task<IActionResult> Login(LoginDto authRequest)
         {
             if (!ModelState.IsValid)
             {
                 return BadRequest(ModelState);
             }
-
-            var result = _authService.Authenticate(authRequest.Email, authRequest.Password);
-            if (!result.Successed)
-            {
-                return BadRequest(result.Message);
-            }
-
+            var (opResult, authResponseModel) = await _authService.Authenticate(authRequest.Email, authRequest.Password);
+            if (!opResult.Successed)
+                {
+                return BadRequest(opResult.Message);
+                }
             var user = _userService.GetByEmail(authRequest.Email);
-
             var userInfo = _mapper.Map<UserInfo>(user);
-            userInfo.Token = result.Message;      
-
+            userInfo.Token = authResponseModel.JwtToken;
+            Response.Cookies.Delete("refreshToken", new CookieOptions { HttpOnly = true });
```

Comment:
You are already deleting cookies here and in `SetTokenCookie` method. Pick one place.

Reply:
Refacored, Deleting cookies in SetTokenCookie method.